cmd/servicelog/post.go

```diff
@@ -51,7 +51,7 @@ var postCmd = &cobra.Command{
 		readFilterFile()      // parse the ocm filters in file provided via '-f' flag
 		readTemplate()        // parse the given JSON template provided via '-t' flag
 
-		if len(args) < 1 && filtersFromFile == "" && clustersFile == "" {
+		if len(args) == 0 && len(filterParams) == 0 && clustersFile == "" {
```

Comment:
filterParams are the query that we use in ocm. The -q parameter directly fills that